docs/Change_Log.md

```diff
@@ -1,5 +1,28 @@
 # Change Log
 
+## API v10
+
+#### May 12, 2022
+
+> info
+> API v10 was made available in [Feb 2022](https://github.com/discord/discord-api-docs/discussions/4510). May 12 just indicates the date for this changelog entry.
+
+- API v8 is now deprecated.
+- `GET /channels/{channel.id}/threads/active` is decomissioned in favor of [`GET/guilds/{guild.id}/threads/active`](#DOCS_RESOURCES_GUILD/list-active-threads).
+- Starting in v10, you must specify the message content intent (using `1 << 15`). See more in the [Gateway Intents documentation](#DOCS_TOPICS_GATEWAY/gateway-intents).
+- To specify a reason for an administrative action in audit logs, apps must now pass the `X-Audit-Log-Reason` rather than the `reason` parameter. Read more in the [Audit Logs documentation](#DOCS_RESOURCES_AUDIT_LOG).
```

Comment:
```suggestion
- The `reason` field has been removed for `PUT /guilds/{guild.id}/bans/{user.id}` and `POST /guilds/{guild.id}/prune` endpoints. The `X-Audit-Log-Reason` header should be used instead. Read more in the [Audit Logs documentation](#DOCS_RESOURCES_AUDIT_LOG).
```

AFAIK the `reason` field parameter only existed for these 2 endpoints.
The original phrasing also looked like that `X-Audit-Log-Reason` was something new, but it has been around for years.